addons/debugger/userscript.js

```diff
@@ -97,7 +97,7 @@ export default async function ({ addon, global, console, msg }) {
     myFlash.block = block;
 
     function _flash() {
-      myFlash.block.svgPath_.style.fill = flashOn ? "#ffff80" : myFlash.colour;
+      myFlash.block.svgPath_.style.fill = flashOn ? "#ff7b26" : myFlash.colour;
```

Comment:
Why?

Reply:
So that it is Scratch Addons-styled not Scratch styled

Reply:
> So that it is Scratch Addons-styled not Scratch styled

It should be Scratch-styled, because the addon runs in the Scratch editor.